pkg/dockerregistry/server/quotarestrictedblobstore.go

```diff
@@ -72,5 +80,44 @@ type quotaRestrictedBlobWriter struct {
 func (bw *quotaRestrictedBlobWriter) Commit(ctx context.Context, provisional distribution.Descriptor) (canonical distribution.Descriptor, err error) {
 	context.GetLogger(ctx).Debug("(*quotaRestrictedBlobWriter).Commit: starting")
 
+	if err := admitBlobWrite(ctx, bw.repo, provisional.Size); err != nil {
+		context.GetLogger(ctx).Error(err.Error())
```

Comment:
You're still double-logging, don't log here at all, since the ones in `admitBlobWrite` are already more descriptive.